resources/user-agents/apps/instagram-app/instagram-app-7-0-on.json

```diff
@@ -1,12 +1,12 @@
 {
-  "division": "Instagram App #MAJORVER#.#MINORVER#",
+  "division": "Instagram App Webview #MAJORVER#.#MINORVER#",
```

Comment:
The browscap build command complained that there were two divisions with the same name (7.0 version), which it looks like there is (since there's a 7.0 version defined in the `instagram-app.json` file which contains differently formatted Instagram patterns than this one. So added the "Webview" text here just to keep it from ignoring the 7.0 version of these patterns (apparently tests don't cover that version at all).